src/native/corehost/hostmisc/pal.h

```diff
@@ -64,7 +64,7 @@
 // degree of compat across their respective releases is usually high.
 //
 // We cannot maintain the same (compat) invariant for linux and thus, we will fallback to using lowest RID-Plaform.
-#if defined(_WIN32)
+#if defined(TARGET_WINDOWS)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
 #define FALLBACK_HOST_RID _X("win10")
```

Comment:
IIRC, is `win` preferred over `win#` as a default/fallback RID now a days?

Reply:
I don't know.. haven't heard anything about that. But I think that for fallback `win` would make sense (the host should not assume windows version without looking for it). That said, fallback on Windows should basically never happen - until there's new windows version which actually raises the major version of the OS, since all of the existing windows versions/architectures are in the RID graph.

Reply:
Ah, ok. I noticed that in release/3.1 branch of coreclr and corefx, multiple project files had `<RuntimeIdentifiers>win7-x64..` etc. but those were changed to version-less `win-x64` during the refactorings in runtime repo. Not sure if there was a decision by infra team but it has aligned windows (`win-<arch>`) with linux (`linux-<arch>`) etc. which is a good thing IMO, as it looks less confusing. 🙂

Reply:
```suggestion
#define FALLBACK_HOST_RID _X("win")
```